presto-main/src/test/java/com/facebook/presto/sql/analyzer/TestAnalyzer.java

```diff
@@ -846,6 +847,26 @@ public void testExplainAnalyze()
         analyze("EXPLAIN ANALYZE SELECT * FROM t1");
     }
 
+    @Test
+    public void testExplainAnalyzeFormatJson()
+    {
+        analyze("EXPLAIN ANALYZE (format JSON) SELECT * FROM t1");
```

Comment:
these just check that we don't fail.  is there a way to test/see what the new output looks like?

Reply:
I added a `deserialize` method on the `JsonRenderedNode` and an associated test inside of `AbstractDistributedTestQueries`